### ISSUE TITLE
Crash in partial perm

### ISSUE THREAD

**Issue by ChrisJefferson** [gap-system/gap#2301]
With kernel debug turned on:

```
gap> PartialPerm([1,2,8],[3,4,1,2]);
gap: src/pperm.c:166: SET_IMG_PPERM: Assertion `DOM_PPERM(f) == ((void *)0) || LEN_PLIST(img) == LEN_PLIST(DOM_PPERM(f))' failed.
```

**Comment by james-d-mitchell** [comment]
Weird, this shouldn't even be called. Fix coming.

**Comment by fingolfin** [comment]
This was fixed by PR #2305
